config.json

```diff
@@ -775,7 +782,13 @@
       "core": false,
       "difficulty": 7,
       "slug": "crypto-square",
-      "topics": null,
+      "topics": [
+        "strings",
+        "text_formatting",
+        "mathematics",
+        "cryptography",
+        "security"
```

Comment:
I would add `lists` as well

Reply:
I think it can be done without using lists also. Strings can be separated using Split() function also.

Reply:
No, the `getPlaintextSegments()` method needs to return a `List` in the [tests](https://github.com/exercism/java/blob/master/exercises/crypto-square/src/test/java/CryptoSquareTest.java) so you have to use lists :)

Reply:
Alright. I'll add it. Thanks for the correction :)

Reply:
No problem :) Thanks for explaining your reasoning :)